arclines/pypit_utils.py

```diff
@@ -6,6 +6,8 @@
 import numpy as np
 import pdb
 
+from arclines import utils as al_utils
```

Comment:
I'll just make a note about this here... I tend to agree with Kyle in [this post](https://github.com/PYPIT/PYPIT/issues/309) about restricting the use of `as` to only when it's absolutely necessary (e.g. different naming of packages/utilities in python 2 vs 3).